### PR TITLE
Implement proper parser for the REPL

### PR DESCRIPTION
**Pull request by jD91mZM2**
Definitely a breaking change. New syntax:

`cmd Arg1 "Argument 2" Arg3 Argument\ 4`, similar to bash
`\n` is now a line break, and can be escaped with `\\n`
`${...}` is now used for substitutions, and can be escaped with `\${...}`.

---

This also proposes using [Just](https://github.com/casey/just) instead of GNU Make so @tbodt doesn't have to feel bad about us abusing the almighty Make by treating it as a simple command runner.

### CODE REVIEW

File: commands.go [hunk 1]
```diff
@@ -1060,51 +1062,44 @@ func msgToString(msg *discordgo.Message) string {
...
-	for i := range args {
-		if strings.Contains(args[i], "{paste}") {
-			clipboardcontent, err := clipboard.ReadAll()
-			if err != nil {
-				stdutil.PrintErr((tl("failed.paste") + err.Error()), nil)
-				return
-			}
-			replacer := strings.NewReplacer("{paste}", clipboardcontent)
-			args[i] = replacer.Replace(args[i])
+func substitute(args string) (string, bool) { // We need a way to escape these.
```
Comment (Mnpn03) [inline commands.go]: Does this still require this comment?

Comment (jD91mZM2) [reply commands.go]: Ah, good catch, thanks!

**Comment by Mnpn03** [comment]
Great, thank you very much!

File: justfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    go install
+
+cross-compile:
+    #!/usr/bin/env sh
+
+    # This requires a script only I, Mnpn, have.
+    # This will not work for you.
+    # pls ignore.
+    cd "$GOPATH/bin"
+    ./Cross\ Compile.sh DiscordConsole discordconsole-team
```
Comment (tbodt) [inline justfile]: Why is this not checked in?

Comment (jD91mZM2) [reply justfile]: I don't have access to the script anymore, but I suppose @Mnpn03 could check it in (although IIRC it largely assumes a bunch of structure)

Comment (Mnpn03) [reply justfile]: Can someone please remind me about this later, thanks!

Comment (jD91mZM2) [reply justfile]: @Mnpn03 Do you know what happened with the guy who invented the door-knocker?

Comment (Mnpn03) [reply justfile]: who’s there?

Comment (jD91mZM2) [reply justfile]: He won the :no_bell: price.

Comment (jD91mZM2) [reply justfile]: :bell: Anyway, this is your friendly reminder to commit the script

Comment (Mnpn03) [reply justfile]: Thanks, I've added it!

**Pull request merged**